pom.xml

```diff
@@ -18,6 +18,11 @@
             <artifactId>maven-resources-plugin</artifactId>
             <version>2.6</version>
         </dependency>
+        <dependency>
```

Comment:
I like that you're using JUnit 5. You shouldn't need both JUnit 5 and 4.12 as dependencies though. IIRC, there's another JUnit 5 jar you can add to your project.